index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Survey Form</title>
```

Comment:
meta tag with viewport value is needed to render webpage better on mobile devices. Please add meta tag with viewport value

Reply:
- [ ] Great job explaining `why` the change is required :+1: